src/Core/src/Platform/Android/LabelExtensions.cs

```diff
@@ -22,5 +23,63 @@ public static void UpdateTextColor(this TextView textView, ILabel label,  Maui.C
 				textView.SetTextColor(textColor.ToNative());
 			}				
 		}
-	}
+
+        public static void UpdateLineBreakMode(this TextView textView, ILabel label)
+        {
+            textView.SetLineBreakMode(label);
+        }
+
+        public static void UpdateMaxLines(this TextView textView, ILabel label)
+        {
+            var maxLines = label.MaxLines;
+
+            if (maxLines == 0)
```

Comment:
is the default value 0?
In original code: `(int)Label.MaxLinesProperty.DefaultValue`

Reply:
If anything, could this be a named constant so we can at least know what this does? `int InfiniteLines = 0;`